### ISSUE TITLE
BUG: TypeError: get_network() got an unexpected keyword argument 'nodes'

### ISSUE THREAD

**Issue by DawnyWu** [HTenkanen/pyrosm#83]
pyrosm version 0.5.3

```python
from pyrosm import OSM, get_data
osm = OSM(get_data("test_pbf"))
nodes, edges = osm.get_network(nodes=True, network_type="driving")
```
TypeError: get_network() got an unexpected keyword argument 'nodes'

**Comment by HTenkanen** [comment]
@DawnyWu Hi! This feature is only available in the new pyrosm version 0.6.0 that I will most probably release later today (sorry you just got in the middle of transition to the new version). If you want to start testing already now, you can install it from the master:
`pip install git+https://github.com/HTenkanen/pyrosm.git@master`

**Comment by HTenkanen** [comment]
New version can now be installed with pip or using conda:
https://pyrosm.readthedocs.io/en/latest/installation.html
